guides/bug_report_templates/action_text_gem.rb

```diff
@@ -0,0 +1,100 @@
+# frozen_string_literal: true
+
+require "bundler/inline"
+
+gemfile(true) do
+  source "https://rubygems.org"
+
+  git_source(:github) { |repo| "https://github.com/#{repo}.git" }
+
+  gem "capybara"
```

Comment:
```suggestion
  gem "capybara"
  gem "puma"
```

Otherwise you will get:

```
/Users/alex/.rvm/gems/ruby-3.2.2/gems/capybara-3.39.2/lib/capybara/registrations/servers.rb:29:in `rescue in block in <top (required)>': Capybara is unable to load `puma` for its server, please add `puma` to your project or specify a different server via something like `Capybara.server = :webrick`. (LoadError)
```

Reply:
I swapped out `:puma` for `:webrick` in an attempt to reduce the number of lines in the template.